client/cypress/integration/dashboard/dashboard_list.js

```diff
@@ -0,0 +1,24 @@
+describe("Dashboard list sort", () => {
+  beforeEach(() => {
+    cy.login();
+  });
+
+  it("creates one dashboard", () => {
+    cy.visit("/dashboards");
+    cy.getByTestId("CreateButton").click();
+    cy.getByTestId("CreateDashboardMenuItem").click();
+    cy.getByTestId("CreateDashboardDialog").within(() => {
+      cy.get("input").type("A Foo Bar");
```

Comment:
Tips: For Form items it's usually good to use `cy.findByLabelText` if the element has a label (which isn't the case), or `cy.findByRole('input', { name: /accessible label/ })`

Since it's just one input and you already have `.within()`, I think you're fine

Reply:
I duplicated this from another test in the suite 👍 Might just leave it.